.github/workflows/release.yml

```diff
@@ -139,3 +109,73 @@ jobs:
            git tag -f ${{ github.event.inputs.releaseVersion }}
            git push --quiet --force
            git push --quiet --force origin ${{ github.event.inputs.releaseVersion }}
+  release-docker:
+    name: Docker release
+    runs-on: 'ubuntu-latest'
+    needs:
+      - release
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.inputs.releaseVersion }}
+          fetch-depth: 1
+          submodules: recursive
+      - name: Docker meta (default)
+        id: meta
+        uses: docker/metadata-action@v3
+        with:
+          images: |
+            ${{ github.repository }}
+            ghcr.io/${{ github.repository }}/${{ github.event.repository.name }}
```

Comment:
I also updated to the current registry at GitHub, see https://docs.github.com/en/packages/working-with-a-github-packages-registry/migrating-to-the-container-registry-from-the-docker-registry for details.